src/scss/main.scss

```diff
@@ -78,37 +100,61 @@
 	height: 24px;
 	vertical-align: baseline;
 	width: 24px;
-	}
+}
 
 .o-teaser__related .o-teaser__related-item .n-syndication-icon {
-		border-radius: 16px;
-		height: 16px;
-		width: 16px;
+	border-radius: 16px;
+	height: 16px;
+	width: 16px;
 }
 
 .hero__heading .n-syndication-icon {
 	vertical-align: baseline;
-	}
+}
 
 // modal
-
 .n-syndication-modal-shadow {
-	@include oOverlayShadow();
-	}
+	background-color: rgba(0, 0, 0, 0.2);
+	position: fixed;
+	height: 150%; // because on some phones scrolling upwards detaches a 100% height overlay from the bottom of the screen
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 10;
+	opacity: 1;
+	transition: opacity 0.3s ease-in-out;
+}
 .n-syndication-modal {
-	@include oOverlay();
+	@include oVisualEffectsShadowContent($elevation: "high");
+	position: fixed;
+	z-index: 10;
+	box-sizing: border-box;
+	opacity: 1;
+	transition: opacity 0.3s ease-in-out;
+	border: $_o-overlay-border-width solid _oOverlayGet("default-border");
+	background: _oOverlayGet("default-background");
 
 	min-height: 160px;
 	max-width: 420px;
 	min-width: 360px;
 	//position: absolute;
 	z-index: 1000;
-	}
+}
 .n-syndication-modal-heading {
-	@include oOverlayHeading();
-	}
+	font-size: 20px;
+	margin: 0;
+	overflow: hidden;
+	background: _oOverlayGet("heading-background");
+	color: _oOverlayGet("heading-text");
```

Comment:
Can we somehow flag this to be refactored to just use o-overlay?

Reply:
i filed this https://github.com/Financial-Times/n-syndication/issues/61